content/docs/releases/enterprise/install/quickstart.mdx

```diff
@@ -1,232 +1,159 @@
-   yum-config-manager --add-repo '/tmp/pomerium-enterprise.repo'
-   yum -q makecache -y --disablerepo='*' --enablerepo='pomerium-enterprise'
-   ```
-
-1. Update refresh and install:
-
-   ```bash
-   yum -y install pomerium-console
-   ```
-
-</TabItem>
-
-</Tabs>
-
-### System Service
-
-Once the package is installed, enable and start the system service:
-
-```bash
-sudo systemctl enable --now pomerium-console
+Learn how to run Pomerium Enterprise using Docker Compose.
+
+## Prerequisites
+
+To complete this guide, you need:
+
+- Pomerium Core
+- A configured identity provider
+- [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/)
+- PostgreSQL
+- Prometheus
+
+This guide runs Pomerium Core and PostgreSQL as Docker containers. If you haven't, complete the [Pomerium Core quickstart](/docs/quickstart) to set up a Pomerium instance that's compatible with this guide.
+
+The Pomerium Enterprise Console uses an embedded Prometheus server to collect and display metrics. While you can [configure metrics](/docs/capabilities/metrics) to use an existing Prometheus server, this guide uses Pomerium’s Prometheus backend.
+
+:::tip **Note**
+
+This guide assumes you've already registered for Pomerium Enterprise and have credentials to access the private Cloudsmith.io Docker registry, and a license key. If you haven't registered for Pomerium Enterprise, [sign up here](https://www.pomerium.com/enterprise-sales/) for a free trial.
+
+:::
+
+### Configure Docker Compose services
+
+Create a file called `docker-compose.yaml` in your project's root folder and add the configuration below:
+
+```yaml title="pomerium/docker-compose.yaml"
+version: '3'
+networks:
+  main: {}
+services:
+  pomerium:
+    image: pomerium/pomerium:latest
+    volumes:
+      - ./config.yaml:/pomerium/config.yaml:ro
+    ports:
+      - 443:443
+    networks:
+      main:
+        aliases:
+          - authenticate.localhost.pomerium.io
+  pomerium_console:
+    networks:
+      main:
+    depends_on:
+      database:
+        condition: service_healthy
+      pomerium:
+        condition: service_started
+    image: docker.cloudsmith.io/pomerium/enterprise/pomerium-console:latest
+    expose:
+      - 8701
+      - 9090
+    environment:
+      - AUDIENCE=console.localhost.pomerium.io
+      - ADMINISTRATORS=admin@example.com
+      - DATABASE_ENCRYPTION_KEY=tXBj4gGDj45m8cW7ehhcy5lRuxsEeNl0X/nnsN5YJPw=
+      - DATABROKER_SERVICE_URL=http://pomerium:5443
+      - SHARED_SECRET=mxGl062SqkrbQKvqG9R2jqHqxq1Oi1BNj2AAeZHNq7c=
+      - DATABASE_URL=postgresql://postgres:postgres@database/postgres?sslmode=disable
+      # - LICENSE_KEY=REPLACE_ME
+      - PROMETHEUS_LISTEN_ADDR=:9090
+      - PROMETHEUS_DATA_DIR=/data
+      - AUTHENTICATE_SERVICE_URL=https://authenticate.localhost.pomerium.io
+      # - SIGNING_KEY=REPLACE_ME
+    volumes:
+      - ./pomerium/metrics:/data:rw
```

Comment:
this would fail trying to mount `./pomerium/metrics` as it does not exist. this may be made a blind volume:

```suggestion
      - metrics-data:/data:rw
```

you just need to also declare that volume at the root. https://docs.docker.com/compose/compose-file/#volumes